lib/benchpark/runtime.py

```diff
@@ -82,7 +82,9 @@ def __init__(self, dest):
         self.spack_location = self.dest / "spack"
 
     def bootstrap(self):
+        print("Hold tight, Benchpark is bootstrapping itself.")
```

Comment:
I notice that e.g. `_install_ramble` has this already, but as a debug print statement:

```
    def _install_ramble(self):
        debug_print(f"Cloning Ramble to {self.ramble_location}")
```

IMO it would be preferable to promote that to a `print()` statement (likewise for spack)